src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -691,6 +696,17 @@ void GraphOptimizer::MergeConvertAndScaleShift(Graph& graph) {
     }
 }
 
+void GraphOptimizer::RemoveConvertF16ToF32(Graph& graph) {
+    auto& graphNodes = graph.GetNodes();
+
+    for (auto parent : graphNodes) {
```

Comment:
From general perspective transformation that just removes some converts from the graph looks weird. Please add explanation that it forces to nodes to have fp16 on directly inputs and allows them to handle precision conversion based on internal logic.

Reply:
added comment, thanks